tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -279,3 +279,237 @@ Feature: copy file
       | permissions |
       | 31          |
       | 17          |
+
+
+  Scenario Outline: Copying a folder within the same space project with different role
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "Project" with the default quota using the GraphApi
+    And user "Alice" has created a folder "folder1" in space "Project"
+    And user "Alice" has created a folder "folder2" in space "Project"
+    And user "Alice" has uploaded a file inside space "Project" with content "some content" to "folder2/demo.txt"
+    And user "Alice" has shared a space "Project" to user "Brian" with role "<role>"
+    When user "Brian" copies folder "folder2" to "folder1/folder2" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Brian" the space "Project" should contain these entries:
+      | folder1/folder2/demo.txt |
+    Examples:
+      | role    |
+      | manager |
+      | editor  |
+
+
+  Scenario: Copying a folder within the same space project with role viewer
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "Project" with the default quota using the GraphApi
+    And user "Alice" has created a folder "folder1" in space "Project"
+    And user "Alice" has created a folder "folder2" in space "Project"
+    And user "Alice" has uploaded a file inside space "Project" with content "some content" to "folder2/demo.txt"
+    And user "Alice" has shared a space "Project" to user "Brian" with role "viewer"
+    When user "Brian" copies folder "folder2" to "folder1/folder2" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "403"
+    And for user "Brian" the space "Project" should not contain these entries:
+      | folder1/folder2/demo.txt |
```

Comment:
IMO These two scenarios can be merged by adding one column 'status_code' to example table.

Reply:
@grgprarup Though the last step of both scenarios is looking for the same resource, one of them expects the resource to exist and the other one doesn't. So if we merge these two scenarios into one, how to check for the resources inside the `Project` space as the expectation of both scenarios are different at the end?:thinking:

Reply:
Got it!